src/main/resources/templates/registration_page.html

```diff
@@ -22,24 +22,24 @@
 <body>
 <main>
     <h1 class="mb-4 text-center">Links</h1>
-    <form action="/registerUser" method="post" enctype="application/x-www-form-urlencoded" autocomplete="on">
+    <form action="/register" method="post" enctype="application/x-www-form-urlencoded" autocomplete="on">
         <div class="mb-3">
             <label for="username" class="form-label">Username</label>
-            <input type="text" class="form-control" id="username" aria-describedby="emailHelp" name="username">
+            <input type="text" class="form-control" id="username" aria-describedby="emailHelp" name="username" value="test1">
```

Comment:
I believe we don't need to commit these changes ("test1", ...)

Reply:
Yeah, we shouldn't. Refactored.
_(how do you catch this things?...)_